sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -595,7 +595,10 @@ def _maybe_add_unified_worker_missing_options(self, options):
       if not debug_options.lookup_experiment('disable_runner_v2'):
         debug_options.add_experiment('beam_fn_api')
         debug_options.add_experiment('use_runner_v2')
-        debug_options.add_experiment('use_portable_job_submission')
+        if (not debug_options.lookup_experiment('use_portable_job_submission')
```

Comment:
`not debug_options.lookup_experiment('use_portable_job_submission')` seems unnecessary since debug_options.add_experiment checks duplicates.

Reply:
Done.